### ISSUE TITLE
Warning: count(): Parameter must be an array or an object that implements Countable

### ISSUE THREAD

**Issue by BernhardWebstudio** [lexik/LexikMaintenanceBundle#79]
## Stack Trace
```
in vendor/lexik/maintenance-bundle/Listener/MaintenanceListener.php at line 184 

at ErrorHandler ->handleError ('2', 'count(): Parameter must be an array or an object that implements Countable', '/.../symfony-www/vendor/lexik/maintenance-bundle/Listener/MaintenanceListener.php', '184', array('event' => object(GetResponseEvent), 'request' => object(Request))) 

at count (null) 
in vendor/lexik/maintenance-bundle/Listener/MaintenanceListener.php at line 184
```
## IMHO
I suspect the problem to be in the server configuration, as otherwise the $ips could not be null – but where exactly?

**Comment by aivus** [comment]
Hi @BernhardWebstudio.
This issue was fixed in #74.

**Comment by BernhardWebstudio** [comment]
Thank you very much!
